onyxia-api/src/main/java/fr/insee/onyxia/api/configuration/CatalogWrapper.java

```diff
@@ -198,13 +209,41 @@ public void setHighlightedCharts(List<String> highlightedCharts) {
     }
 
     public List<String> getIncludeKeywords() {
+        if (includeKeywords == null) return new ArrayList<>();
         return includeKeywords;
     }
 
     public void setIncludeKeywords(List<String> includeKeywords) {
         this.includeKeywords = includeKeywords;
     }
 
+    public void setExcludeKeywords(List<String> excludeKeywords) {
+        this.excludeKeywords = excludeKeywords;
+    }
+
+    public List<String> getExcludeKeywords() {
+        if (excludeKeywords == null) return new ArrayList<>();
```

Comment:
I'm not a huge fan of this pattern of returning different references. In some cases it may leads to odd bugs if the caller then adds items to the returned list for example. 
In one case the item would be added to the "real" list but if the list is null then the item would be added to a transient list and will not show up in a next get call.
Wouldn't it suffise to have default empty lists for each variable ? IIRC all objectmappers / json / yaml deserializers has been configured to set an empty list and not a null list so we should be safe

Reply:
@olevitt Yes I agree, good point, I'll make that change 👍